main.go

```diff
@@ -70,6 +71,7 @@ func runPrometheusServer(metricsAddr string, tlsOptions common.SSPTLSOptions) er
 	setupLog.Info("Starting Prometheus metrics endpoint server with TLS")
 	metrics.Registry.MustRegister(common_templates.CommonTemplatesRestored)
 	metrics.Registry.MustRegister(common.SSPOperatorReconcileSucceeded)
+	sspMetrics.SetupMetrics()
```

Comment:
Can't we reuse the existing metrics components? SSP already exposes metrics, so why introduce a new library?

Reply:
following https://github.com/kubevirt/community/pull/219, we can already start moving forward towards these new ideas